applications/fluid_structure_interaction/pressure_wave/input.json

```diff
@@ -17,6 +17,7 @@
         "AbsTol": "1.e-9",
         "RelTol": "1.e-3",
         "OmegaInit": "0.3",
+        "UseExtrapolation": "true",
```

Comment:
probably not a bug, but we should remove this again, or use the enum instead

Reply:
ah sry forgot that

Reply:
I ran it now just to be sure:
pressure wave example
extrapolation: 6.78 iterations on avg
I will update #574 now and let you knwo when its ready!